src/olympia/bandwagon/tests/test_models.py

```diff
@@ -61,12 +61,16 @@ def test_auto_uuid(self):
         assert isinstance(c.uuid, uuid.UUID)
 
     def test_collection_meta(self):
-        c = Collection.objects.create(author=self.user)
-        assert c.addon_count == 0
-        c.add_addon(Addon.objects.all()[0])
+        some_time_ago = self.days_ago(442)
+        collection = Collection.objects.create(author=self.user)
+        collection.update(modified=some_time_ago)
+        assert collection.addon_count == 0
+        self.assertCloseToNow(collection.modified, now=some_time_ago)
```

Comment:
Nit: This is a cool function, but why is it needed? Wouldn't `collection.modified` be equal to `some_time_ago` at this point?

Reply:
It would, but I'm being paranoid because `modified` is a bit special. I initially used `Collection.objects.create(author=self.user, modified=some_time_ago)` but this does *not* work as you'd think - `modified` is set to more recent date.